### ISSUE TITLE
Access to Github, Dockerhub, AWS, Kubernetes

### ISSUE THREAD

**Issue by nonsense** [testground/testground#1289]
Make sure everyone has access to relevant infrastructure

- [x] give access to all repos under https://github.com/testground org
- [x] ~give access to DockerHub https://hub.docker.com/u/iptestground org~ let's migrate to AWS ECR
- [x] give access to AWS account (so that folks can create prod/test k8s clusters)
  - [x] Bloxico
  - [x] EngRes Productivity
- [ ] give access to individual k8s cluster
  - [ ] Bloxico
  - [ ] EngRes Productivity

**Comment by BigLep** [comment]
Dockerhub:  
- Need DockerHub identities

AWS
- It's dedicated (mostly) to TestGround
- Will create IAM user per active maintainer
 - Create a Bloxico admin account (can create lower permissions accounts as needed)
 - PL EngRes individuals need it too

Individual k8s cluster
* https://ci.testground.ipfs.team/tasks
* kops
* On AWS
* There is outdated (most likely) docs on this
* Does various setup (EFS, etc.)

**Comment by laurentsenta** [comment]
Hey team, is there anything blocking this task (esp the EngRes Productivity part), I could help with?

(waiting for AWS access to take a shot at https://github.com/testground/testground/issues/1290)

**Comment by nonsense** [comment]
> Hey team, is there anything blocking this task (esp the EngRes Productivity part), I could help with?

Done. If anyone else needs access, ping me!

**Comment by nonsense** [comment]
Folks, I forgot to mention, we also have a private Github repository, where we keep playbooks that are specific to https://ci.testground.ipfs.team/

---

Feel free to ping me on DM so that I give you access to review them and update them as needed:

https://github.com/protocol/infra-testground/tree/master/taas/k8s

---

cc @brdji @laurentsenta @galargh

**Comment by BigLep** [comment]
Lets manage access using github management files.  It should be
self-service for folks to submit PRs.

**Comment by galargh** [comment]
@BigLep The repo is in `protocol` which was put on hold for now.

**Comment by nonsense** [comment]
@galargh I don't have permission to move repos from `protocol`. I just copied over the repo for now, in order to unblock you folks:

https://github.com/testground/pl-infra-testground

Let me know if this works for you.

**Comment by galargh** [comment]
Given https://github.com/testground/testground/issues/1289#issuecomment-1067800480 let's assume it's done. Let's open new issues if we're missing some access.
